src/environments/environment.test.ts

```diff
@@ -1,4 +1,4 @@
 export const environment = {
   production: false,
-  couchPort: '5984'
+  couchAddress: 'http://127.0.0.1:5984/'
```

Comment:
@paulbert so for test we access couchdb on 5984 in all others 2200 ... right?

Reply:
@paulbert by the way how to switch between environment with `ng`?

Reply:
@dogi since test passed seems the config used is the `environment.ts` instead of `environment.test.ts` what do you think @paulbert ?